class.jetpack.php

```diff
@@ -1819,15 +1819,16 @@ public static function load_private() {
 		}
 	}
 
+	/**
+	 * Adds a message to the 'At a Glance' dashboard widget.
+	 *
+	 * @param string $content Content of 'At A Glance' wp-admin dashboard widget.
+	 */
 	public static function add_public_dashboard_glance_items( $content ) {
+		wp_enqueue_style( 'private', plugins_url( 'modules/private/private.css', __FILE__ ), array(), '20190521' );
```

Comment:
Do you think it would make sense, since there is so little CSS in there, to use `wp_add_inline_style` instead and add that CSS below a CSS file enqueued by core for the dashboard widgets, if that's possible?

Reply:
Thanks again @jeherve ! I've implemented your suggestion in [ba35111](https://github.com/Automattic/jetpack/pull/10333/commits/ba35111ec9c24543ba8693c795586102fbbef405).